src/ucs/sys/math.h

```diff
@@ -24,18 +24,18 @@ BEGIN_C_DECLS
 #define UCS_GBYTE    (1ull << 30)
 #define UCS_TBYTE    (1ull << 40)
 
-#define ucs_min(a, b) \
+#define ucs_min(_a, _b) \
 ({ \
-    typeof(a) _a = (a);  \
-    typeof(b) _b = (b);  \
-    _a < _b ? _a : _b;   \
+    typeof(_a) __a = (_a); \
```

Comment:
typically we use var names without `_` prefix inside macro

Reply:
Done